spring-petclinic-microservices/terraform/setup_env.sh

```diff
@@ -1,39 +1,22 @@
-#!/usr/bin/env bash
-
-# ==== Resource Group ====
-export RESOURCE_GROUP=resource-group-name # customize this
-export LOCATION=SouthCentralUS  #customize this
-export COSMOSDB_NAME=mycosmosdbaccname  # customize this
-export REDIS_NAME=myredisname #customize this
-export KEYVAULT_NAME=myend2endkv #customize this
-export APP_NAME_FOR_KEYVAULT=myappforkeyvault #customize this
-
-# ==== Create ResouceGroup ====
-echo "Creating ResouceGroup, it may take a few minutes"
-az group create --name ${RESOURCE_GROUP} \
-        --location ${LOCATION}
-
-# ==== Create CosmosDB Account ====
-echo "Creating CosmosDB Account, it may take a few minutes"
-az cosmosdb create --name $COSMOSDB_NAME --resource-group $RESOURCE_GROUP
-COSMOS_KEYS=$(az cosmosdb keys list --name $COSMOSDB_NAME --resource-group $RESOURCE_GROUP --type keys)
-COSMOS_PRIMARY_KEY=$(echo $COSMOS_KEYS | jq -r .primaryMasterKey)
-COSMOS_SECONDARY_KEY=$(echo $COSMOS_KEYS | jq -r .secondaryMasterKey)
-COSMOSDB_URI=$(az cosmosdb  show --name $COSMOSDB_NAME --resource-group $RESOURCE_GROUP | jq -r .documentEndpoint)
+export AZURE_KEYVAULT_URI=$(terraform -chdir=./terraform output -raw keyvault_url)
+export RESOURCE_GROUP=$(terraform -chdir=./terraform output -raw resource_group_name)
+export KEYVAULT_NAME=$(terraform -chdir=./terraform output -raw azurerm_key_vault_account)
+export COSMOSDB_URI=$(terraform -chdir=./terraform output -raw azure_cosmos_endpoint)
+export COSMOS_PRIMARY_KEY=$(terraform -chdir=./terraform output -raw azure_cosmos_key)
+export LOCATION=$(terraform -chdir=./terraform output -raw location)
+export REDIS_NAME=redis-petclinic-samples #customize this
+export APP_NAME_FOR_KEYVAULT=app-petclinic-samples #customize this
 
 # ==== Create Redis Cache Account ====
 echo "Creating Redis Cache Account, it may take a few minutes"
```

Comment:
it takes me too much time to create redis accout,

OS: MAC OS
```
➜  ~ az version
{
  "azure-cli": "2.33.1",
  "azure-cli-core": "2.33.1",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {}
}
```


please use the latest version of az cli to check this.